base/shell.jl

```diff
@@ -81,7 +81,7 @@ function shell_parse(raw::AbstractString, interp::Bool)
                     if done(s,k)
                         error("unterminated double quote")
                     end
-                    if s[k] == '"' || s[k] == '$'
+                    if s[k] == '"' || s[k] == '$' || s[k] == '\\'
```

Comment:
Sorry. Yes, this is intentional (see the "parse" in the title). Currently, quoted backslashes in double quotes are not correctly interpreted; they result in two backslashes: The string `"\\"` is parsed to yield `\\` instead of `\`.